openstack/nova/compute.sls

```diff
@@ -100,6 +108,7 @@ my_ip compute paranoia:
     - require:
```

Comment:
This is also a bad default on a compute - each node can identify itself by this ip at times.